pdns/pdns_recursor.cc

```diff
@@ -2389,6 +2367,11 @@ static bool checkForCacheHit(bool qnameParsed, unsigned int tag, const string& d
     }
     g_stats.avgLatencyUsec = (1.0 - 1.0 / g_latencyStatSize) * g_stats.avgLatencyUsec + 0.0; // we assume 0 usec
     g_stats.avgLatencyOursUsec = (1.0 - 1.0 / g_latencyStatSize) * g_stats.avgLatencyOursUsec + 0.0; // we assume 0 usec
+#if 0
+    // XXX changes behaviour compared to old code!
+    g_stats.answers(0);
+    g_stats.ourtime(0);
+#endif
```

Comment:
I'm afraid this change will cause a lot of confusion about a "latency regression" :-) Perhaps we could add a new metric but keep the existing one (but that means double work)? Otherwise I think we need to write some "impossible to miss" upgrade note :)

Reply:
That's why I `#if 0`ed that part... in the future, the plan is to add configurable histograms. Those will be an addition, not a replacement iof the current metrics, as not to disturb.